modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/form_fields.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview::V1
+      class FormFields
+        def first_name
+          'F[0].#subform[2].VeteransFirstName[0]'
+        end
+
+        def middle_initial
+          'F[0].#subform[2].VeteransMiddleInitial1[0]'
```

Comment:
You could probably pull out F[0].#subform[2] to a separate method to dry up the code a bit

Reply:
Good point! I'm going to do that.

Reply:
Oh and pulling out F[0].#subform[3] would be another refactor too 👍